src/components/accessories/admin/operations/editOperation/EditOperation.tsx

```diff
@@ -0,0 +1,40 @@
+import { useTranslation } from "react-i18next";
+import OperationForm from "../operationForm/OperationForm";
+import React from "react";
+import { getInitialFields } from "../operationForm/consts";
+import { useDispatch, useSelector } from "react-redux";
+import { OperationDTO } from "../../../../../generated";
+import { IApiResponse } from "../../../../../state/types";
+import { updateOperation } from "../../../../../state/operations/actions";
+import { IState } from "../../../../../types";
+import { Navigate, useLocation, useParams } from "react-router";
+import { PATHS } from "../../../../../consts";
+
+export const EditOperation = () => {
+  const dispatch = useDispatch();
+  const { t } = useTranslation();
+  const { state }: { state: OperationDTO | undefined } = useLocation();
+  const { id } = useParams();
+  const update = useSelector<IState, IApiResponse<OperationDTO>>(
+    (state) => state.operations.update
+  );
+
+  const handleSubmit = (value: OperationDTO) => {
+    dispatch(updateOperation({ code: id ?? "", operationDTO: value }));
+  };
+
+  if (state?.code !== id) {
+    return <Navigate to={PATHS.operations} />;

```

Comment:
Again, `PATHS.admin_operations` but it depends on the conversation in https://github.com/informatici/openhospital-ui/pull/594/files#r1619964530